src/Heatmap/GMUHeatMapInterpolationPoints.m

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2020 Google Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#if !defined(__has_feature) || !__has_feature(objc_arc)
+#error "This file requires ARC support."
+#endif
+
+#import "GoogleMapsUtils-Swift.h"
+
+@implementation HeatMapInterpolationPoints : NSObject  {
```

Comment:
Is this a test file?

Reply:
Yeah. I deleted it in the Xcode project, though... not sure why it's still there. In any case, it should be gone now.